README.md

```diff
@@ -88,6 +88,12 @@ is configured in the frontmatter for each docs page. The magic happens in in `pa
 
 For pages _except_ docs and for some other tags, look at changing [`next-seo.config.js`](./next-seo.config.js).
 
+### Task: Adding a Comment in Documentation
+
+Sometimes you'll want to add a comment which is only for documentation maintainers; in regular markdown you'd use HTML style comments: `<!-- my comment -->`
```

Comment:
Can I encourage you to write this of the form "while you might be inclined to...". Starting with a note about how to do something only to write in a later paragraph "don't do the thing we mentioned in the previous paragraph" is counterproductive at best and really closer to setting someone up to fail.

Reply:
Really good suggestion yeah, thanks, will reword this :+1: